.github/workflows/ci-slow.yml

```diff
@@ -260,27 +293,22 @@ jobs:
     secrets: inherit
   custom-ubuntu-integration-test:
     if: github.event.pull_request.draft == false
-    needs: run-slow-ci-label-is-set
+    needs: [run-slow-ci-label-is-set, custom-ubuntu-unit-test]
     strategy:
       matrix:
-        os: [arc-runner-set]
-        python-version: ['3.9', '3.10', '3.11', '3.12']
-        test_environment:
-          - default
-          - docker-server-docker-orchestrator-mysql
-          - docker-server-docker-orchestrator-mariadb
+        os: [ubuntu-latest]
+        python-version: ['3.9', '3.10', '3.12']
```

Comment:
what happened to 3.11?

Reply:
it's used in fast CI i thought no need to repeat since that happens first anyways